core/webapp/pom.xml

```diff
@@ -15,7 +16,13 @@
     </parent>
 
     <name>Vitro Languages for Web App CORE</name>
-
+    
+    <properties>
```

Comment:
If we update the top-level pom.xml with these additional properties, then all `<modules>` will inherit them, no?
https://github.com/vivo-project/Vitro-languages/blob/sprint-i18n/pom.xml#L55-L63

Reply:
I'am not shure. I'll run some tests and I'll come back with the solution.